builtin/logical/pki/acme_challenge_engine_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package pki
+
+import (
+	"fmt"
+	"net/netip"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// Test_encodeIdentifierForHTTP01Challenge validates the encoding behaviors of our identifiers
+// for the HTTP01 challenge. Basically properly encode the identifier for an HTTP request.
+func Test_encodeIdentifierForHTTP01Challenge(t *testing.T) {
+	ip, _ := netip.ParseAddr("2001:db8::68")
+	fmt.Printf(ip.StringExpanded())
```

Comment:
Could it be you forgot to clean up this println?

Reply:
Oops yes indeed thanks! Will clean it up

Reply:
Removed in https://github.com/hashicorp/vault/pull/28718/commits/c9837ab533a7ab71160dc1286c2dc413e07fa244